blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/AbstractPipelineImpl.java

```diff
@@ -231,6 +233,22 @@ public Link getLink() {
         };
     }
 
+    @Override
+    public List<Object> getParameters() {
+        return getParameterDefinitions(job);
+    }
+
+    public static List<Object> getParameterDefinitions(Job job){
```

Comment:
Although this was already merged, PMD should allow introspection for the parameters, since there's a well defined-ish format.